mobile/src/Student/screens/StudentName/index.js

```diff
@@ -12,16 +12,27 @@ const StudentName = ({ navigation, gameSession, setTeamInfo }) => {
     const [lastName, setLastName] = useState("")
     const firstNameTextRef = useRef(null)
     const lastNameTextRef = useRef(null)
+    const [showErrorText, setShowErrorText] = useState(false)
 
     onNameSubmit = () => {
-        if (!firstName && firstNameTextRef) {
+        /*if (!firstName && firstNameTextRef) {
             firstNameTextRef.focus()
             return
         }
 
         if (!lastName && lastNameTextRef) {
             lastNameTextRef.focus()
             return
+        }*/
```

Comment:
Always go with removing the code instead of commenting it. This helps to keep the code clean

Reply:
Yes, will delete before next push! Didn't realize I hadn't deleted it